### PR TITLE
feat: shop manager documentation

### PR DESCRIPTION
**Pull request by MohanNarayana**
Signed-off-by: Mohan Narayana <mohan.narayana@mailchimp.com>

Shop manager structured as documentation. Duplicate content(existing on /dev) removed.

### CODE REVIEW

**Comment by brent-hoover** [comment]
@MohanNarayana Is this a duplicate of the other Shop Manager PR? If so can you close one or the other

File: docs/create-and-manage-a-shop.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Creating and managing a shop
+
+## The basics
+Congratulations on a successful installation of Open Commerce. Now, you can access the Admin Portal at http://localhost:4080/
+![Admin](_assets/01-admin-clean.png). The Open Commerce Admin is the user interface used by administrators and shop managers to set up the required shop settings and manage operations of the shop.
+
+## Create a user
+To create a user, click the profile icon in the top right corner of the screen and select **Create Account**.
+
+
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: @MohanNarayana remove extra line

File: docs/create-and-manage-a-shop.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+![Switch shop](_assets/02-admin-switch-shop.png)
+
+Once you have created a user and set up a shop, the admin portal will look like this.
+![Admin](_assets/02-admin-user-shop-setup.png)
+
+## Accounts
+
+You'll find account management and settings options by clicking Accounts in the side menu of the Open Commerce Admin portal.
+
+Within Accounts you can access the following options::
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: @MohanNarayana remove extra colon

File: docs/create-and-manage-a-shop.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Staff Members: View and modify the accounts involved with shop management and the different permission groups to which they belong. The first account created is populated under Owner. You can also create additional groups and invite users here.
+- Customers: View customers of the shop..
+- Invitations: View the outstanding sent invitations. Once the user accepts an invitation it is no longer shown here.
+
+### Create Group
+
+To create a new group, click Create Group under Staff Members, and add the following information in the pop-up modal:
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md:38]: remove comma after 'and'

File: docs/create-and-manage-a-shop.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Don't forget to save your changes!
+
+### Invite Staff Member
+
+You can invite people to be a part of your shop management by clicking Invite Staff Member. You'll need to enter their Name and Email ID, and then you can select the groups to which you want to add this member.
+
+Once you've successfully sent the invitation, the recipient should receive an email.
+
+> Note: You'll need to set up an email service to send out invitations. See [Email Configuration](75-email-configuration.md) for more information.
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: don't think this link is out there. maybe pointing to the transactional API config that @Akarshit did a while ago?

File: docs/create-and-manage-a-shop.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## Payments
+
+To set up Payments, go to Settings > Payments in the admin dashboard. Click to enable or disable a payment method. Enabled payment methods are visible to the customer at checkout.
+You can use the IOU Example payment method to test your store before enabling your actual payment method, like Stripe or another supported payment processor.
+Open Commerce includes a [Stripe](https://stripe.com/) plugin for payments. The new plugin is [SCA complient](https://stripe.com/docs/strong-customer-authentication).
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: typo: compliant

File: docs/create-and-manage-a-shop.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+You can invite people to be a part of your shop management by clicking Invite Staff Member. You'll need to enter their Name and Email ID, and then you can select the groups to which you want to add this member.
+
+Once you've successfully sent the invitation, the recipient should receive an email.
+
+> Note: You'll need to set up an email service to send out invitations. See [Email Configuration](75-email-configuration.md) for more information.
+
+## Payments
+
+To set up Payments, go to Settings > Payments in the admin dashboard. Click to enable or disable a payment method. Enabled payment methods are visible to the customer at checkout.
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: Settings > payments should have the same bold format as similar instructions below

File: docs/create-and-manage-a-shop.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+You can use the IOU Example payment method to test your store before enabling your actual payment method, like Stripe or another supported payment processor.
+Open Commerce includes a [Stripe](https://stripe.com/) plugin for payments. The new plugin is [SCA complient](https://stripe.com/docs/strong-customer-authentication).
+
+## Taxes
+
+To set up Taxes, go to **Settings > Taxes** in the admin dashboard.
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md:62]: like here

File: docs/create-and-manage-a-shop.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+- **Enabled**: Select this to make the option visible on the storefront.
+
+Finally, click **Save changes** to save your new settings.
+
+Shipping methods enabled at storefront on checkout.
+![Storefront Shipping at Checkout](_assets/74-storefront-shipping-checkout.png)
+
+## Address validation
+
+You can find settings for the for Address Validation under **Settings > Address Validation** on the admin dashboard.
```
Comment (delagroove) [inline docs/create-and-manage-a-shop.md]: extra for

**Review by delagroove — COMMENTED** [review]
found some issues

File: docs/create-and-manage-a-shop.md (hunk 3 above)
Comment (MohanNarayana) [reply docs/create-and-manage-a-shop.md:38]: This line is a recommendation from Missy

File: docs/create-and-manage-a-shop.md (hunk 4 above)
Comment (MohanNarayana) [reply docs/create-and-manage-a-shop.md]: That doc is not merged. Pointing to Sending Emails in /dev.

**Review by delagroove — APPROVED** [review]

**Comment by brent-hoover** [comment]
Closing to be moved to a different place

**Pull request closed without merging**